content/en/docs/collector/deployment/gateway/index.md

```diff
@@ -198,6 +199,30 @@ The load-balancing exporter emits metrics including
 that you can use for health and performance monitoring of the OTLP endpoint
 collector.
 
+## Typical Gateway deployment architecture
+
+The following diagram shows a typical deployment architecture for a gateway
+![gateway](otel-gateway-arch.svg)
+
+There are a few limitations in running the OTel collector in gateway mode.
+
+- Few receivers need to be unique per host instance. Running multiple instances
```

Comment:
> Few receivers need to be unique per host instance. 

I don't understand what that sentence means.

Reply:
Updated, please take a look.